tests/testapp/tests/test_mixin.py

```diff
@@ -339,9 +338,9 @@ def test_comparison_state_should_reset_after_save(self):
         account = UserAccount.objects.create(**data)
         account.first_name = "Maggie"
         self.assertTrue(account.has_changed("first_name"))
-        with capture_on_commit_callbacks(execute=True) as callbacks:
+        with self.captureOnCommitCallbacks(execute=True) as callbacks:
             account.save()
-        self.assertEquals(len(callbacks), 1, msg="Only the _reset_initial_state should be in the on_commit callbacks")
```

Comment:
`assertEquals` was deprecated on Python 3.2 and finally removed in 3.12: https://docs.python.org/3.12/whatsnew/3.12.html#id3